apps/web/components/dashboard/tags/TagPill.tsx

```diff
@@ -62,27 +67,25 @@ export function TagPill({
     },
   );
 
-  return (
-    <Draggable
-      key={id}
-      axis="both"
-      onStart={dragAndDropFunction.handleDragStart}
-      onStop={dragAndDropFunction.handleDragEnd}
-      disabled={!isDraggable}
-      defaultClassNameDragging={"position-relative z-10 pointer-events-none"}
-      position={{ x: 0, y: 0 }}
+  const pill = (
+    <div
+      className="group relative flex"
+      onMouseOver={handleMouseOver}
+      onFocus={handleMouseOver}
+      onMouseOut={handleMouseOut}
+      onBlur={handleMouseOut}
     >
-      <div className="group relative flex">
-        <Link
-          className={
-            "flex gap-2 rounded-md border border-border bg-background px-2 py-1 text-foreground hover:bg-foreground hover:text-background"
-          }
-          href={`/dashboard/tags/${id}`}
-          data-id={id}
-        >
-          {name} <Separator orientation="vertical" /> {count}
-        </Link>
+      <Link
+        className={
+          "flex gap-2 rounded-md border border-border bg-background px-2 py-1 text-foreground hover:bg-foreground hover:text-background"
+        }
+        href={`/dashboard/tags/${id}`}
+        data-id={id}
+      >
+        {name} <Separator orientation="vertical" /> {count}
+      </Link>
 
+      {isHovered && (
```

Comment:
This seems to break the deletion dialog. Maybe we can have just one dialog in the page, and trigger it with that button.